crates/rpc/rpc-eth-types/src/cache/mod.rs

```diff
@@ -512,6 +534,7 @@ enum CacheAction<B: Block, R> {
     GetBlockWithSenders { block_hash: B256, response_tx: BlockWithSendersResponseSender<B> },
     GetHeader { block_hash: B256, response_tx: HeaderResponseSender<B::Header> },
     GetReceipts { block_hash: B256, response_tx: ReceiptsResponseSender<R> },
+    GetCachedBlock { block_hash: B256, response_tx: CachedBlockResponseSender<B> },
```

Comment:
the name here, or at least docs, better reflect that this doesn't enqueue the block request like `GetBlockWithSenders` does